src/Kunstmaan/MediaBundle/Helper/ManipulateImageService.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Helper;
+
+use Imagine\Filter\Basic\Crop;
+use Kunstmaan\MediaBundle\Entity\CroppableMediaLink;
+use Kunstmaan\MediaBundle\Entity\Media;
+use Kunstmaan\UtilitiesBundle\Helper\Slugifier;
+use Liip\ImagineBundle\Service\FilterService;
+
+class ManipulateImageService
+{
+    /** @var Slugifier */
+    private $slugifier;
+
+    /** @var FilterService */
+    private $filterService;
+
+    public function __construct(Slugifier $slugifier, FilterService $filterService)
+    {
+        $this->slugifier = $slugifier;
+        $this->filterService = $filterService;
+    }
+
+    public function getFocusPointClass(CroppableMediaLink $croppableMediaLink, string $view = ''): string
+    {
+        $class = '';
+        if ($croppableMediaLink->getRunTimeConfig() !== null) {
```

Comment:
I would changes to method so it does more early returns. This will result in less nested if's and more readable code. Example:

```
if (null === $croppableMediaLink->getRunTimeConfig()) {
    return '';
}
```

Reply:
This method never returns ' '. it will always return a useable url to the image even if there is no configuration present. for that reason i wrote it without this early return.

Reply:
But it actually does no? If the runtime config is null you won't enter the if and the `$class = '';` value is returned

Reply:
yes you are right i need to do the empty check. thanks i missed that

Reply:
yes sorry i was mistaken